docs/guides/ci-integration.md

```diff
@@ -73,7 +77,13 @@ The following environments are known to require additional settings:
 * GitHub Actions: requires `FORCE_COLOR=1`
 * Jenkins: requires `NO_COLOR=1`
 
-## Step 5: Run the build
+### Adjust target name column
+
+The CLI defaults to a right-aligned, 20-character width to display the name of the target that the log originated from. When a target name exceeds 20 characters, the name is truncated and the last 20 characters are used.
+
+ * `TARGET_PADDING=n` will set the column to the width of `n` characters.
+ * `FULL_TARGET=1` will always print the full target name, and nothing more. This may result in unaligned logs.
```

Comment:
I would prefix these with `EARTHLY_` like we did with most env vars. (We didn't prefix `NO_COLOR` and `FORCE_COLOR` as they are more standard env var names)